### PR TITLE
✨Custom training options

### LINKED ISSUE #601: 📄 Events - Custom Training Options
https://www.ssw.com.au/ssw/Events/Training/Training-Options.aspx
```[tasklist]
### Tasks
- [x] Migrate content from old page.
- [x] Add new banner
- [x] Update megamenu link
- [x] Add redirect to front door
```

### PR DESCRIPTION
**Pull request by KristenHu**
Fixes #601 - Migrate events page: Custom Training Options

Affected routes:

-`events/customized-courses`

Add done video, screenshots

### CODE REVIEW

**Review by bradystroud — CHANGES_REQUESTED** [review]
image
These testimonials should use the testimonials component

FYI the url in your description gives a 404, it should be `events/customized-courses`

**Comment by bradystroud** [comment]
Hey @KristenHu 
Is this one ready for review?
Seems like my requested changes haven't been fixed yet

Please make this a draft PR if its not ready for review

**Comment by KristenHu** [comment]
> Hey @KristenHu Is this one ready for review? Seems like my requested changes haven't been fixed yet
> 
> Please make this a draft PR if its not ready for review

Hey @bradystroud ,
Yes, It's ready for review now. Ken and I just got test pass for the banner of the page this morning, I've pushed it to the PR.

As per our conversation in Teams, we won't use testimonials component here currently, so there's nothing needs to be fixed😊
image

File: public/images/background/SSWTraining.png
Comment (bradystroud) [inline public/images/background/SSWTraining.png:1]: This file isn't used, can you remove?

File: content/events/customized-courses.mdx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
...ogan, Database Services Manager for SSW and a Microsoft Regional Director, the seminars will guide you through the latest Microsoft technologies and help you improve your development processes.\n\nView and sort our index of SSW\_[events led by industry experts](https://www.ssw.com.au/ssw/Events/Training/Courses.aspx).\n\nYou can also check our\_[Schedule](https://www.ssw.com.au/ssw/Training/Schedule.aspx)\_to see all upcoming events.\n\n## Power Sessions and Customized Training\n\nPower sessions are developer training presentations focusing on specific subject and are generally 1-2 hours long. They are packed with practical tips on .NET, SQL, Silverlight, Visual Studio and more. Perfect for when you're trying to dig deeper on a particular subject or just want to learn something new. Sessions are rotated on various events and roadshows around the world all year round, including SSW's own\_[Tech Breakfast](https://www.ssw.com.au/ssw/Events/SSWTechBreakfast.aspx)\_and\_[.NET User Group](https://www.ssw.com.au/ssw/NETUG/Sydney.aspx). Meet with fellow developers and discuss pretty much anything.\n\n[Have a look at all Power Sessions](https://www.ssw.com.au/ssw/training/Sessions.aspx)\n"
```
Comment (bradystroud) [inline content/events/customized-courses.mdx]:
```suggestion
    content: "## Keep up with the latest developments on Microsoft Technologies\n\n✅Hands-on training\n\n✅High quality material\n\n✅Real world examples\n\nWe believe that everyone should have access to expert advice. SSW provides high quality hands-on developer training so you can stay in front. Our instructors have substantial experience and speak at development conferences around the world and material is presented in a workshop type format featuring real-world scenarios.\n\n## The Perfect Training\n\nSSW invites you to maximise your investment in Microsoft technologies. Led by technical expert Adam Cogan, Database Services Manager for SSW and a Microsoft Regional Director, the seminars will guide you through the latest Microsoft technologies and help you improve your development processes.\n\nGet a head start by attending SSW's series of seminars and conferences led by industry experts.\n\n✅Hours are customizable and flexible\n\n✅Our trainers can come to you\n\n✅Wide range of topics on Microsoft technologies\n\nWhether you want to improve your skills, project or resume, SSW can help you achieve your goals. Our courses cover a broad range of topics in various formats from full-day workshops to evening courses spanning across ten weeks. Small group size guarantees close individual attention given by our knowledgeable and passionate trainers.\n\n> Many thanks to the fine folks at SSW... for coming to our offices and providing the training we needed specifically.\n> — PHILLIP CLARKSON\n\n> \"Eric and SSW have been fantastic. They brought significant experience and pragmatic advice to BuzzNumbers. As a result of their engagement we have implemented a full Project Management process that has dramatically improved team productivity and software quality. SSW had the technical know-how and the direct business implementation experience to deliver on-time results that were above our expectations\"\n> — NICK HOLMES\n\n## Developer Training Courses and Events\n\nMaximise your investment in Microsoft technologies. Led by technical expert Adam Cogan, Database Services Manager for SSW and a Microsoft Regional Director, the seminars will guide you through the latest Microsoft technologies and help you improve your development processes.\n\nView and sort our index of SSW\_[events led by industry experts](https://www.ssw.com.au/ssw/Events/Training/Courses.aspx).\n\nYou can also check our\_[Schedule](https://www.ssw.com.au/ssw/Training/Schedule.aspx)\_to see all upcoming events.\n\n## Power Sessions and Customized Training\n\nPower sessions are developer training presentations focusing on specific subject and are generally 1-2 hours long. They are packed with practical tips on .NET, SQL, Silverlight, Visual Studio and more. Perfect for when you're trying to dig deeper on a particular subject or just want to learn something new. Sessions are rotated on various events and roadshows around the world all year round, including SSW's own\_[Tech Breakfast](https://www.ssw.com.au/ssw/Events/SSWTechBreakfast.aspx)\_and\_[.NET User Group](https://www.ssw.com.au/ssw/NETUG/Sydney.aspx). Meet with fellow developers and discuss pretty much anything.\n\n[Have a look at all Power Sessions](https://www.ssw.com.au/ssw/training/Sessions.aspx)\n"
```

**Review by bradystroud — CHANGES_REQUESTED** [review]

File: public/images/background/SSWTraining.png
Comment (KristenHu) [reply public/images/background/SSWTraining.png:1]: **Done**😊

**Comment by KristenHu** [comment]
As per my conversation with @bradystroud , this PR will be closed since there are too much unknown changes in .tina folder, and the new PR has been created: https://github.com/SSWConsulting/SSW.Website/pull/1361

**Pull request closed without merging**